### DISCUSSION TITLE
What does the old hymns option do?

### DISCUSSION THREAD

**Discussion by fiapps** [DivinumOfficium/divinum-officium#2990] [Developers corner]
I'm considering adding support for the old hymns option to my app, but I'd like to understand better what it does. Does it use the pre-Urban VIII form of the hymns? From the code, it looks like it checks for a `HymnusM`, and uses that, if available. The name suggests that it is the monastic form of the hymn. Is that always the same as the Breviarium Romanum hymn before Urban VIII's revisions?

I looked at the [help](https://divinumofficium.com/www/horas/Help/help.html), but it hasn't been updated to describe this option or "Hide verse numbers."

**Comment by mbab** [comment]
The name HymnusM is misleading there. For non Monastic versions using this option means pre-Urban hymns.
